tokio-util/CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# 0.6.5 (March 20, 2021)
+
+### Fixed
+
+- chore: fix clippy warnings in newer versions ([#3588])
```

Comment:
Well, this change has no effect on the user, so strictly, this doesn't need to be mentioned in the changelog.

Reply:
Good catch. I've removed that line.